app/config.local.js

```diff
@@ -29,3 +34,4 @@ kubernetes: "dev-mode"
 };
 
 window.MOCK_ORIGIN_SERVICES = false;
+window.MOCK_SERVICES_DATA = true;
```

Comment:
Ok, understand how we want to mock things going forward, will work towards that.  Ok if we merge this and I follow up with another PR?